apps/content/migrations/0033_add_asset_slug.py

```
@@ -0,0 +1,20 @@
+        ("content", "0032_populate_asset_external_fields"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="asset",
+            name="slug",
+            field=models.SlugField(
+                allow_unicode=True, default="", help_text="URL-friendly slug for the asset", unique=True
+            ),
+        ),
+    ]
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)